notary/server/Makefile

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Gramine contributors
+# SPDX-License-Identifier: BSD-3-Clause
+
+ARCH_LIBDIR ?= /lib/$(shell $(CC) -dumpmachine)
+
+SELF_EXE = target/release/notary-server
```

Comment:
the `target/release/notary-server` file is actually located in the parent folder: `notary/target/release`, instead of `notary/server/target/release` — will this affect anything?

Reply:
the latest repo reorg has shifted the target folder to the top level of the repo